src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -135,6 +152,42 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('read_enabled', help='Indicates whether read operation is allowed.', arg_type=get_three_state_flag())
         c.argument('write_enabled', help='Indicates whether write or delete operation is allowed.', arg_type=get_three_state_flag())
 
+    with self.argument_context('acr manifest') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr', validator=validate_registry_name)
+        c.argument('top', type=int, help='Limit the number of items in the results.')
+        c.argument('orderby', help='Order the items in the results. Default to alphabetical order of names.', arg_type=get_enum_type(['time_asc', 'time_desc']))
+        c.argument('delete_enabled', help='Indicates whether delete operation is allowed.', arg_type=get_three_state_flag())
+        c.argument('list_enabled', help='Indicates whether this item shows in list operation results.', arg_type=get_three_state_flag())
+        c.argument('read_enabled', help='Indicates whether read operation is allowed.', arg_type=get_three_state_flag())
+        c.argument('write_enabled', help='Indicates whether write or delete operation is allowed.', arg_type=get_three_state_flag())
+        c.argument('repository', help='The name of the repository.', options_list=['--name', '-n'], validator=validate_repository)
+        c.argument('manifest_id', help="The name of the artifact. May include a tag in the format 'name:tag' or digest in the format 'name@digest'.", options_list=['--name', '-n'])
+
+    # Positional arguments must be specified on each individual command, they cannot be assigned to a command group
+    with self.argument_context('acr manifest show') as c:
+        c.positional('ID', arg_type=manifest_id_type)
+        c.argument('raw_output', help='Output the raw manifest text with no formatting.', options_list=['--raw'], action='store_true')
+
+    with self.argument_context('acr manifest list') as c:
+        c.positional('ID', arg_type=repo_id_type)
+
+    with self.argument_context('acr manifest delete') as c:
+        c.positional('ID', arg_type=manifest_id_type)
+
+    with self.argument_context('acr manifest list-referrers') as c:
+        c.positional('ID', arg_type=manifest_id_type)
+        c.argument('artifact_type', help='Filter referrers based on artifact type.')
+        c.argument('recursive', help='Recursively include referrer artifacts.', action='store_true')
+
+    with self.argument_context('acr manifest metadata show') as c:
+        c.positional('ID', arg_type=manifest_id_type)
```

Comment:
This problem is caused by the positional argument you defined is not in the argument list of the method executed by the command．
Therefore, the method you call should not be the Python SDK method, but the custom method. Then you need to declare the `ID` parameter in the custom method